### ISSUE TITLE
[a11y] Issue 32: Screen reader should announce outer card container before reading card content

### ISSUE THREAD

**Issue by kk-slider18** [NewJerseyStyle/polis#128]
**Description:**
When reading a comment, the screen reader should first announce the outer card container (group) before reading the card content. Currently, the card content is read directly, which affects context understanding.

**Component/Area:** Open Participation Page

**Source:** Screen Reader

**Affected Elements:** Comments

**Comment by cplkake** [comment]
Classified a P1 issue because it affects how users understand the context of the comment but is not a critical priority issue

**Comment by Natasha1110** [comment]
This may be caused by DOM order. Screen readers follow the structure of your HTML (DOM), to determine the reading order. So, we may start by looking at the HTML and making sure it is structured within this flow of all content within a "group" container in the HTML.

**Comment by YumiChen** [comment]
Just to double check, is this referring to this section? ( I was looking at [this page](https://[pol.is/5esrbenwxs?ui_lang=en](https://pol.is/5esrbenwxs?ui_lang=en)))

![Image](https://github.com/user-attachments/assets/fd75a035-2bb3-4293-8510-583c112cc1e5)
